src/main/java/io/nats/streaming/StreamingConnectionFactory.java

```diff
@@ -58,7 +63,7 @@ public StreamingConnection createConnection() throws IOException, InterruptedExc
     Options options() {
         return new Options.Builder().connectWait(connectTimeout).pubAckWait(ackTimeout)
                 .discoverPrefix(discoverPrefix).maxPubAcksInFlight(maxPubAcksInFlight)
-                .natsConn(natsConn).natsUrl(natsUrl).build();
+                .natsConn(natsConn).natsUrl(natsUrl).connectionListener(connListener).errorListener(errListener).build();
```

Comment:
Do these new init stubs require a test as well?

Reply:
should be tested in the options tests